plugins/a11ychecker.md

```diff
@@ -161,91 +295,212 @@ If `a11ychecker_allow_decorative_images` is set to `false`, the Accessibility Ch
+
+**Type:** `Function`
+
+**Default Value:** `undefined`
+
+#### Example: Using `a11ychecker_issue_url_callback`
+
+This example shows how to change the link for the "Click for more info" button (![help icon - a question mark inside a circle]({{site.baseurl}}/images/icons/help.svg)) on the Accessibility Checker dialog to point to anchors at `www.example.com/tinymce/a11ychecker/more_info`.
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_issue_url_callback: function (ruleId) {
+    return `https://www.example.com/tinymce/a11ychecker/more_info#${ruleId}`
+  };
 });
 ```
 
 ### `a11ychecker_level`
 
 This configuration option sets the [WCAG level](https://www.w3.org/TR/WCAG20/#conformance) to use when checking issues.
 
-For example, the "Text must have a contrast ratio of at least ..." rule when using **AA** will trigger when the contrast ratio is less than 4.5:1, however when using **AAA** the rule will trigger when the ratio is less than 7.0:1.
+For example, the "Text must have a contrast ratio of at least ..." rule:
+
+* When using **AA**, Accessibility Checker will check for a contrast ratio of less than 4.5:1.
+* When using **AAA**, Accessibility Checker will check for a contrast ratio of less than 7.0:1.
 
 **Type:** `String`
 
 **Default value:** `aa`
 
 **Possible Values:** `a`, `aa`, `aaa`
 
-#### Example: a11ychecker_level
+#### Example: Using `a11ychecker_level`
 
 ```js
 tinymce.init({
-    selector: 'textarea',
-    plugins: 'a11ychecker',
-    toolbar: 'a11ycheck',
-    a11ychecker_level: 'aaa'
+  selector: 'textarea',
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_level: 'aaa'
 });
 ```
 
 {% include misc/plugin-toolbar-button-id-boilerplate.md %}
 
 {% include misc/plugin-menu-item-id-boilerplate.md %}
 
-## API
+## APIs
 
-Accessibility Checker exposes couple of methods that can be called directly.
+The {{pluginname}} plugin provides the following APIs.
 
 ### `toggleaudit()`
 
-Triggers accessibility dialog with the results of the audit and options to correct the problems, if any.
+Opens the accessibility checker dialog with the results of the audit and options to correct the problems, if any.
+
+#### Example: Using `toggleaudit()`
 
-#### Example: toggleaudit()
 ```js
 editor.plugins.a11ychecker.toggleaudit();
 ```
 
 ### `getReport()`
 
-Conducts accessibility audit and reports about the results without triggering the dialog. The report represents an array of issues, each of which has details about:
+Conducts accessibility audit and reports the results without triggering the dialog. The report represents an array of issues, with the following details:
```

Comment:
```suggestion
Conducts accessibility audit and reports the results without triggering the dialog. The report represents an array of issues, each with the following details:
```

Reply:
```suggestion
Conducts accessibility audit and reports the results without triggering the dialog. The report produced is an array of objects, with each object representing an issue and contains the following details:
```
@ashfordneil ?

Reply:
containing?

Reply:
no, then it sounds like the issue contains those details, not the object.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1699/commits/36bc08703715c08ebab95f6f83d6b60cc1daac3e